static/js/components/AnonymousMenu.js

```diff
@@ -22,15 +22,17 @@ const AnonymousMenu = ({ useScreenOverlay }: Props) => {
     <ul className="menu-holder">
       <li {...listItemProps || {}}>
         <MixedLink
+          id="login"
           dest={routes.login.begin}
           className="simple"
-          aria-label="Login"
+          aria-label="Sign In"
         >
           Sign In
         </MixedLink>
       </li>
       <li {...listItemProps || {}}>
         <MixedLink
+          id="create_account"
```

Comment:
Nit: I think we want to stick with camel case for id/name attributes

```suggestion
          id="createAccount"
```

Reply:
@gsidebo, Thanks for the suggestion, I've converted it to the camel case and in the process, I did a quick refactoring in the AnonymousMenu as well, similar to what I did in UserMenu in this PR. 

Here is the commit https://github.com/mitodl/mitxonline/pull/168/commits/22dc55ab142fd8bcd13603238b3793e6e1dc3de5, please let me know if you'd like any changes or this looks good as well.